### PR TITLE
feat(VBtnGroup/VBottomNavigation): Add baseColor prop

### LINKED ISSUE #17042: [Feature Request] Support `active-color` prop on `v-bottom-navigation` similar to `v-list`
### Problem to solve
Prevents need to write model bindings on `v-bottom-navigation`

### Proposed solution
Add a prop to `v-bottom-navigation` component to set the applied color when the component is in an active state.

### PR DESCRIPTION
**Pull request by yuwu9145**
fixes #17042

## Description

Copy "baseColor" logic from VListItem

## Markup:

```vue
<template>
  <v-card class="mx-auto" max-width="300">
    <v-list :items="items" color="green" base-color="red"></v-list>
  </v-card>
  <br>
  <br>
  <br>
  <v-card class="mx-auto" max-width="300">
    <v-btn-toggle color="green" :base-color="baseColor" v-model="toggle_exclusive">
      <v-btn>
        <v-icon>mdi-format-align-left</v-icon>
      </v-btn>

      <v-btn>
        <v-icon>mdi-format-align-center</v-icon>
      </v-btn>

      <v-btn>
        <v-icon>mdi-format-align-right</v-icon>
      </v-btn>

      <v-btn>
        <v-icon>mdi-format-align-justify</v-icon>
      </v-btn>
    </v-btn-toggle>
    
  </v-card>
  <v-btn
    @click="() => baseColor = 'yellow'"
  >
    change basecolor to yellow
  </v-btn>
</template>

<script>
  export default {
    data: () => ({
      toggle_exclusive: 2,
      baseColor: 'red',
      items: [
        {
          title: 'Item #1',
          value: 1,
        },
        {
          title: 'Item #2',
          value: 2,
        },
        {
          title: 'Item #3',
          value: 3,
        },
      ],
    }),
  }
</script>


```

### CODE REVIEW

File: packages/vuetify/src/components/VBtnGroup/VBtnGroup.tsx [hunk 1]
```diff
@@ -44,6 +45,7 @@ export const VBtnGroup = genericComponent()({
     provideDefaults({
       VBtn: {
         height: 'auto',
+        baseColor: toRef(props, 'baseColor'),
```
Comment (KaelWD) [inline packages/vuetify/src/components/VBtnGroup/VBtnGroup.tsx:48]: Same thing on bottom nav.

Comment (yuwu9145) [reply packages/vuetify/src/components/VBtnGroup/VBtnGroup.tsx:48]: thanks, done

**Comment by MajesticPotatoe** [comment]
is it a bug fix or a feature?

**Comment by yuwu9145** [comment]
> is it a bug fix or a feature?

Small new feature

**Review by johnleider — APPROVED** [review]

**Pull request merged**